scripts/split_identify_fasta.py

```diff
@@ -11,35 +11,44 @@
 parser.add_argument("--Ns", type = int, default = 5000, help = "Number of Ns to split by [5000]")
 
 args = parser.parse_args()
+def read_fasta(fp):
```

Comment:
You might want to leave a comment as to how you're parsing this fasta. It's arguably overkill, but maybe just a comment so others immediately see what the function does, and how you're parsing the data. Nothing too crazy.

My philosophy is, whatever you could do now which stops confused github issues in the future, please do it :) You can imagine not everyone using this script is 100% familiar with python syntax.

Reply:
It's not super easy to explain this. But I added a comment that points the interested person in the right direction.

Reply:
> It's not super easy to explain this.

You don't need to necessarily over explain. 

I think commenting what the basic python does in the function works too (for Alex to follow 🤫)